package.json

```diff
@@ -29,6 +29,7 @@
     "@commitlint/config-conventional": "17.4.4",
     "@ls-lint/ls-lint": "2.0.1",
     "@typescript-eslint/eslint-plugin": "5.48.1",
+    "@typescript-eslint/parser": "5.62.0",
```

Comment:
You really need it?

Reply:
I don’t know why but without it, lint doesn’t work, and when you try to run "npm run lint:js" it can’t find the parser module. I tried to install the parser globally but lint didn’t see it, on stackoverflow I found  that the last lint versions can’t use plugins globally

Reply:
Interesting, OK, let's try with installed pkg